src/input.js

```diff
@@ -11,40 +11,71 @@ export const mydata = () => {
     if (e.key === 'Enter') {
       const totaltasks = create();
       const task = {
-        check: true,
-        des: document.querySelector('.main').value,
-        index: totaltasks.length,
+        description: document.querySelector('.main').value,
+        completed: false,
+        index: totaltasks.length + 1,
```

Comment:
As requested by the previous code reviewer,   the index should b set to the value of the new array length  😄 (i.e. if a 5th task is added to the list, that task's index should equal 5).  Kindly update that 👍 

![Capture](https://github.com/MayPyone/to-do-list/assets/80458199/881410a3-24c4-44da-acfa-426b60da44ff)